src/core/hle/service/am/am.cpp

```diff
@@ -1674,6 +1767,31 @@ void Module::Interface::GetMetaDataFromCia(Kernel::HLERequestContext& ctx) {
     rb.PushMappedBuffer(output_buffer);
 }
 
+void Module::Interface::BeginImportTicket(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp(ctx);
+
+    // Create our TicketFile handle for the app to write to
+    const FileSys::Path path = {};
+    auto file =
+        std::make_shared<Service::FS::File>(am->kernel, std::make_unique<TicketFile>(), path);
```

Comment:
shouldn't this work as well?
```suggestion
        std::make_shared<Service::FS::File>(am->kernel, std::make_unique<TicketFile>(), {}});
```